WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -136,6 +136,15 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
         ZendeskUtils.sharedInstance.showNewRequestIfPossible(from: sourceViewController, with: sourceTag)
     }
 
+    @objc func testSignupEpilogue(from sourceVC: UIViewController) {
```

Comment:
I assume this should be removed?

Reply:
RIGHT.

I've been using that to test the epilogue during development 😆